### DISCUSSION TITLE
Is using temp and humidity offset applied linearly?

### DISCUSSION THREAD

**Discussion by gurrier** [pvvx/ATC_MiThermometer#421] [General]
I've got a few LYWSD03MMC B1.6 sensors around the house and after acquiring a dehumidifier, I decided to try to align them through a crude calibration process.
Is the offset in the firmware config for temp and humidity applied linearly right across the scale as a fixed offset?
Does that work well if I just place each sensor side by side and then apply a consistent offset at current room temperature with a known good measurement e.g. the dehumidifier for example?

**Comment by pvvx** [comment]
To do this, you will need to place the sensors in a special cabinet. On a table or shelf they will always show different values due to natural convection.
